init.sql

```diff
@@ -20,278 +20,3 @@ SET row_security = off;
 -- Name: enum_transcripts_status; Type: TYPE; Schema: public; Owner: postgres
 --
 
-CREATE TYPE public.enum_transcripts_status AS ENUM (
```

Comment:
Are we dropping the seed data here? I think we'll want this for local dev

Reply:
yeah that was data for the old schema. Am going to include data with the new fields